### PR TITLE
Created portfolio

### PR DESCRIPTION
**Pull request by Alexr16**
Created README file
Created HTML file
Created CSS file
Web page responsive created.
Portfolio: setup and mobile version skeleton ending.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <li class="social">
+                <a href="#"><img src="images/mail.svg" alt="email icon"></a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </div>
+    </nav>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please wrap this section with a `header` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <img src="images/twitter.svg" alt="twitter icon">
+            <img src="images/medium.svg" alt="medium icon">
+        </div>
+        <div class="headline-description">
+            <h1>Hey There. I’m Jorge</h1>
+            <h4>I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
```
Comment (CollinsTatang) [inline index.html:57]: Great job here respecting HTML Semantic

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    </nav>
+    <section class="headline">
+        <div class="headline-icons">
+            <img src="images/github.svg" alt="github icon">
+            <img src="images/linkedinicon.svg" alt="linkedin icon">
+            <img src="images/angellist.svg" alt="angel icon">
+            <img src="images/twitter.svg" alt="twitter icon">
+            <img src="images/medium.svg" alt="medium icon">
+        </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please remove the `img` tag and group your icons with `ul` tag and `li` per icon, each icon has a link  with `a` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            <h1>Hey There. I’m Jorge</h1>
+            <h4>I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="headline-ilustration">
+            <img src="images/Illustration-header.svg" alt="ilustration image">
+        </div>
+    </section>
```
Comment (CollinsTatang) [inline index.html:61]: The general design is very good! All background images, margins, and details are really similar to Figma's! Awesome job

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Alexr16 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
## Highlights
- No linter errors ✔️ 
- PR is descriptive ✔️
- Gitflow flow was correctly used ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alexr16** [comment]
Section `<header>` added.
img tag removed and group in an `<ul>.`
`<li>` tag used to list image icons and linked with `<a>` tag.

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Alexr16, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**